client/lang/src/en.json

```diff
@@ -2,17 +2,21 @@
   "ElementArchiveAction.CONFIRM_DELETE": "Are you sure you want to send this block to the archive?",
   "ElementArchiveAction.CONFIRM_DELETE_AND_UNPUBLISH": "Warning: This block will be unpublished before being sent to the archive. Are you sure you want to proceed?",
   "ElementArchiveAction.ARCHIVE": "Archive",
+  "ElementArchiveAction.ARCHIVE_DISABLED": "Archive, insufficient permissions",
```

Comment:
`ARCHIVE_DISABLED` key name does not imply any relevance with lacking permission.
(`disabled` conventionally means the feature is off altogether, not something perms related).
Feels like we might use a different name that could be more relevant to the use case (`canArchive` permission check)